great_expectations/core/expectation_suite.py

```diff
@@ -937,6 +943,33 @@ def _get_expectations_by_domain_using_accessor_method(
 
         return expectation_configurations_by_domain
 
+    def render(self) -> None:
+        """
+        Renders content using the atomic prescriptive renderer for each expectation configuration associated with
+           this ExpectationSuite to ExpectationConfiguration.rendered_content.
+        """
+        for expectation_configuration in self.expectations:
+            inline_renderer_config: "InlineRendererConfig" = {
+                "class_name": "InlineRenderer",
+                "render_object": expectation_configuration,
+            }
```

Comment:
This is a dictionary right? Should we have this type hint?

Reply:
We had a previous chain about how this should be a `TypedDict` and this method is required due to circular imports.